src/test/TestUtils.h

```diff
@@ -10,28 +10,13 @@
 namespace stellar
 {
 
-class LedgerManagerEditableVersion : public LedgerManagerImpl
-{
-  public:
-    LedgerManagerEditableVersion(Application& app, uint32_t baseFee);
-
-    uint32_t getCurrentLedgerVersion() const override;
-    void setCurrentLedgerVersion(uint32_t currentLedgerVersion);
-
-    void startNewLedger() override;
+Application::pointer
+createTestApplication(VirtualClock& clock, Config const& cfg);
 
-  private:
-    uint32_t mBaseFee;
-    uint32_t mCurrentLedgerVersion;
-};
-
-class ApplicationEditableVersion : public ApplicationImpl
+namespace testutil
```

Comment:
we already have txtest namespace for testing transaction, maybe use testutil for that as well?

Reply:
I think it is better to keep `txtest` and `testutil` separate. If there are functions which are not specific to transaction tests in `txtest` then we should move those to `testutil`. But having a namespace which has the utility functions for a given set of tests is a reasonable way to organize things in my opinion. I have even followed this approach in the tests for invariants that I have been writing, where there is a namespace for some of the test-specific functionality.

Reply:
OK, that is reasonable.